app/src/main/java/com/andrew/tasky/presentation/event_detail/EventDetailFragment.kt

```diff
@@ -0,0 +1,723 @@
+package com.andrew.tasky.presentation.event_detail
+
+import android.graphics.Paint
+import androidx.lifecycle.ViewModelProvider
+import android.os.Bundle
+import android.view.*
+import android.widget.PopupMenu
+import androidx.activity.result.ActivityResultCallback
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.core.content.res.ResourcesCompat
+import androidx.fragment.app.Fragment
+import androidx.core.view.isVisible
+import androidx.fragment.app.setFragmentResult
+import androidx.fragment.app.setFragmentResultListener
+import androidx.navigation.NavController
+import androidx.navigation.Navigation
+import androidx.navigation.fragment.navArgs
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.andrew.tasky.R
+import com.andrew.tasky.databinding.CvReminderLayoutBinding
+import com.andrew.tasky.databinding.CvTimeDateSelectorBinding
+import com.andrew.tasky.databinding.FragmentEventDetailBinding
+import com.andrew.tasky.domain.AgendaItem
+import com.andrew.tasky.domain.AgendaItems
+import com.andrew.tasky.domain.Attendee
+import com.andrew.tasky.domain.AttendeeType
+import com.andrew.tasky.presentation.adapters.AttendeeItemAdapter
+import com.andrew.tasky.presentation.adapters.PhotoItemAdapter
+import com.andrew.tasky.presentation.dialogs.DatePickerFragment
+import com.andrew.tasky.presentation.dialogs.DeleteConfirmationDialogFragment
+import com.andrew.tasky.presentation.dialogs.TimePickerFragment
+import com.andrew.tasky.util.*
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+import java.time.format.DateTimeFormatter
+
+class EventDetailFragment : Fragment(R.layout.fragment_event_detail) {
+
+    private val currentDate = LocalDateTime.now()
+        .format(DateTimeFormatter.ofPattern("dd MMMM yyyy")).uppercase()
+
+    private lateinit var viewModel: EventDetailViewModel
```

Comment:
I'd prefer this to initialize your VM:
`private val viewModel: EventDetailViewModel by viewModels()`